src/components/namelist/nameList.js

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect } from "react";
+import { get } from "../../service/apiClient";
+import "./nameList.css";
+import ProfileCircle from "../profileCircle";
+
+function NameList({ inputText }) {
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
+    const getUserInfo = async () => {
+      const res = await get(`users`);
```

Comment:
Should be using double-quotes and not back-ticks, since we are not interpolating anything.